### PR TITLE
Add in `pepembed` README

### PR DESCRIPTION
**Pull request by nleroy917**
So, right now, this is just that one README, but I'd be interested to see what people have to say about either potentially breaking it up or even just adding more sections.

### CODE REVIEW

**Comment by nsheff** [comment]
I think it should live under PEPhub, not have its own tab at the top level.

File: docs/pepembed/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# pepembed
+
+## Overview
+Command line interface and python package for computing text-embeddings of sample metadata stored in [pephub](https://github.com/pepkit/pephub) for search-and-retrieval tasks. The purpose of this package is to handle the long-running job of downloading projects inside pephub, mining any relevant metadata from them, and then computing a rich text embedding on that data and upserting it into a vector database. We use [qdrant](https://qdrant.tech/) as our vector database for its performance and simplicity and payload capabilities.
```
Comment (nsheff) [inline docs/pepembed/README.md:4]:
```suggestion
PEPembed is a Python package for computing text-embeddings of sample metadata stored in [pephub](https://github.com/pepkit/pephub) for search-and-retrieval tasks. It provides both a CLI and a Python API. It handles the long-running job of downloading projects inside pephub, mining any relevant metadata from them, computing a rich text embedding on that data, and finally upserting it into a vector database. We use [qdrant](https://qdrant.tech/) as our vector database for its performance and simplicity and payload capabilities.
```

File: docs/pepembed/README.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+**4. Insert Embeddings:**  
+Finally, we insert the embeddings into a [qdrant](https://qdrant.tech/) instance. qdrant is a **vector database** that is designed to store embeddings as first-class data types as well as supporting native graph-based indexing of these embeddings. The allows for near-instant search and retrieval of nearest embeddings neighbors given a new embedding (say an encoded search query on a web application). qdrant supports arming the embeddings with a [**payload**](https://qdrant.tech/documentation/payload/) where we store basic information on that PEP like registry path, row id, and its description.
+
+## Install and Run
+While simple to install and run, `pepembed` requires lots of information to function. There are three key aspects: 1) The pephub instance, 2) the qdrant instance, and 3) the keywords. Ensure the following before running the `cli`:
```
Comment (nsheff) [inline docs/pepembed/README.md:51]: I think generally you want an empty line after a header in markdown

File: docs/pepembed/README.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+### Run
+
+```console
+pepembed \
+  --keywords-file keywords.txt \
+  --postgres-host $POSTGRES_HOST \
+  --postgres-user $POSTGRES_USER \
+  --postgres-password $POSTGRES_PASSWORD \
+  --postgres-db $POSTGRES_DB \
+```
```
Comment (nsheff) [inline docs/pepembed/README.md:102]: what is `pepquery` referenced in the image?

**Review by nsheff — APPROVED** [review]

File: docs/pepembed/README.md (hunk 3 above)
Comment (nleroy917) [reply docs/pepembed/README.md:102]: It _was_ an imagined microservice that converts text to embeddings...

But now that's just pephub. So I can change that

Comment (nleroy917) [reply docs/pepembed/README.md:102]: That image is a `png` for some reason, so I need to change it to SVG and edit it to change that

**Pull request merged**